test/acceptance/testdata/NonStructuredCustomResourceDefinition/customresourcedefinition.tf

```diff
@@ -0,0 +1,27 @@
+provider "kubernetes-alpha" {
+}
+
+resource "kubernetes_manifest" "test" {
+  provider = kubernetes-alpha
+
+  manifest = {
+    apiVersion = "apiextensions.k8s.io/v1beta1"
+    kind       = "CustomResourceDefinition"
+    metadata = {
+      name = "${var.plural}.${var.group}"
+    }
+    spec = {
```

Comment:
I would be inclined to explicitly set `preserveUnknownFields` to `true` even though that's the default, if only just to clearly express the purpose of the test.